src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -17,13 +17,18 @@
 use Symfony\Contracts\EventDispatcher\Event as BaseEvent;
 
 /**
+ * @template TSubject
+ *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  * @author Carlos Pereira De Amorim <carlos@shauri.fr>
  */
 class Event extends BaseEvent
 {
     protected $context;
+    /**
+     * @var TSubject
+     */
```

Comment:
We should rather annotate the constructor parameter.

Reply:
Agree...